spikingjelly/activation_based/triton_kernel/triton_utils.py

```
@@ -137,15 +139,37 @@ def _check_pytorch_version(version_s: str = "2.4") -> bool:
-        except BaseException:
-            pass  # ignore the errors
+    for cache_dir in _triton_codegen_cache_dirs():
+        if not cache_dir.is_dir():
+            continue
+        for pattern in ("*.py", "*.py.tmp", "*.tmp"):
+            for f in cache_dir.glob(pattern):
+                try:
+                    f.unlink()
+                except FileNotFoundError:
+                    pass
+                except Exception as exc:
```

Ruff (0.15.11)

[warning] Do not catch blind exception: `Exception`

(BLE001)